examples/all-clusters-app/all-clusters-common/src/rvc-modes.cpp

```diff
@@ -41,12 +42,15 @@ void RvcRunModeDelegate::HandleChangeToMode(uint8_t NewMode, ModeBase::Commands:
 {
     uint8_t currentMode = mInstance->GetCurrentMode();
 
-    // Our business logic states that we can only switch into a running mode from the idle state.
-    if (NewMode != RvcRunMode::ModeIdle && currentMode != RvcRunMode::ModeIdle)
+    if (!gRvcRunModeInstance->HasFeature(static_cast<ModeBase::Feature>(RvcRunMode::Feature::kDirectModeChange)))
```

Comment:
@bzbarsky-apple are you planning to make similar updates to the rvc-app example application?

Reply:
@plauric I would like to, but that one seemed more complicated and I need a bit more time to sort out what the right thing to do there is.